straxen/plugins/veto_pulse_processing.py

```diff
@@ -21,6 +21,11 @@
         default=20, track=True,
         help='Minimum hit amplitude in ADC counts above baseline. '
              'Specify as a tuple of length n_nveto_pmts, or a number.'),
+    strax.Option(
+        'min_samples_alt_baseline_nv',
+        default=50000, track=True,
```

Comment:
Is this activated or deactivated? It looks like it's on right, perhaps add a value (None, False) to the docstring to signify it's off?

Reply:
I changed it to a value which cannot be stratified, but you are right None or False would be more clean. Let me change it.